src/app/file-explorer.js

```diff
@@ -250,10 +235,11 @@ function fileExplorer (appAPI, files) {
     element.parentElement.replaceChild(el, element)
     element = el
   }
-
-  element.events = events
-  element.api = api
-  return element
+  
+  var events = new EventManager()
+  var api = { events: events }
+  container.appendChild(element)
+  return api
```

Comment:
ok, maybe. that's why i wrote the long comment above.

In JavaScript, the `new` operator and `this` is considered bad style and should be avoided.
But it exists for traditional reasons.
Here is the longer explanation: https://github.com/ethereum/browser-solidity/pull/557#discussion_r118080066

...anyway, i can do it and add it as you wish.